conda-forge.yml

```diff
@@ -3,6 +3,7 @@ azure:
   max_parallel: 20
 bot:
   abi_migration_branches:
+  - 14.x
```

Comment:
JFYI, we'll still need to create that branch; I can do this once we merge.

Reply:
Already did this as I expect we will merge this PR quite soon(ish).

Reply:
> Already did this as I expect we will merge this PR quite soon(ish).

FWIW, let's please only create branches once the PR gets merged. I've now backported #1292 from `main` to `14.x`, but that's kind of annoying to keep track of.